examples/torch_disc/torch_disc_train.sh

```diff
@@ -4,26 +4,37 @@
 #   https://huggingface.co/bert-base-cased/resolve/main/pytorch_model.bin \
 #   -O bert-base-cased.bin
 
-rm -rf dump && mkdir -p dump
-#export LTC_IR_DEBUG=1
-export LTC_SAVE_TENSORS_FILE=dump/ltc_ir.txt
-export LTC_SAVE_TENSORS_FMT=backend
-
-#export APPLY_PENDING_GRAPH_MATERIALIZE=on
-#export LTC_EMIT_STEPLOG=1
-#train_script="pytorch/xla_spawn.py --num_cores 1"
-#train_script="-m torch.distributed.launch --nproc_per_node 1"
-#export TORCH_DISC_LTC_DISABLE_DISC=true
-export TORCH_DISC_LTC_DUMP_TS_GRAPH_DATA=true
-export PYTHONPATH=$(pwd)/../../src
-export CUDA_VISIBLE_DEVICES=1
-export LTC_DISC_CUDA=1
-#export BENCHMARK_ENABLE_NVPROF=1
-#export BENCHMARK_DISABLE_TORCH_DISC=1
-export LD_LIBRARY_PATH=/usr/local/cuda-11.0/targets/x86_64-linux/lib/:$LD_LIBRARY_PATH
-#nvprof --profile-from-start off python $train_script \
-python $train_script \
-  ../pytorch/text-classification/run_glue.py \
+function set_default_env() {
```

Comment:
Nit: shall we split training scripts and README.md into another pull request?

Reply:
I think it's okay, the README described the usage of the script, they are related.